lib/passport-verify.ts

```diff
@@ -1,39 +1,11 @@
 /**
  * Entry point for the passport-verify npm module.
  *
- * Exports [[createStrategy]] and [[createResponseHandler]] which can be used as follows:
- *
- * ```
- * const passportVerify = require('passport-verify')
- * passport.use(passportVerify.createStrategy(
- *   verifyServiceProviderHost,
- *   function createUser (user) { },
- *   function verifyUser (user) { },
- *   function saveRequestId (requestId, request) { },
- *   function loadRequestId (request) { }
- * ))
- * ```
- *
- * ```
- * // route for authenticating a user
- * app.post('/verify/start', passport.authenticate('verify'))
- *
- * // route for handling a callback from verify
- * app.post('/verify/response', (req, res, next) => (
- *   passport.authenticate('verify',
- *     passportVerify.createResponseHandler({
- *       onMatch: user => {},
- *       onCreateUser: user => {},
- *       onAuthnFailed: failure => {},
- *       onError: error => {},
- *     })
- *   )
- * )(req, res, next))
```

Comment:
We got rid of this lot because it's no longer comprehensive now that passport-verify has to support both matching and non-matching.  There were only two other choices: either extend the comment to provide similar information for the non-matching equivalents of these two functions (which would make ti very long and probably very confusing), or split all of this stuff into two files.

I actually quite like the latter idea, but it then raises the question of what we call the two files, and opens a can of worms about how to reference them in other documentation.  Reducing the scope of the comment feels like a more pragmatic solution.